tests/test_run_plan.py

```diff
@@ -118,6 +123,41 @@ class TestFullRunPlan:
     run_plan = RunPlan.create(
         all_known_projects={execution_1.project, execution_2.project}, plan=full_plan
     )
+    test_run_json = Path("tests/.mpyl/run_plan.json")
+
+    @patch("src.mpyl.run_plan.RUN_PLAN_JSON_FILE", test_run_json)
+    def test_write_to_json(self):
+        full_plan = {
+            build_stage: {execution_1, execution_2},
+            test_stage: {execution_1},
+            deploy_stage: {execution_2, cached_execution},
+        }
+        run_plan = RunPlan.create(
+            all_known_projects={execution_1.project},
```

Comment:
It doesn't matter too much for this test, but for correctness' sake:
```suggestion
            all_known_projects={execution_1.project, execution_2.project, cached_execution.project},
```

Reply:
I did this at first, but then I found out that this field is only used for the namespace logic in the chart generation. So it has 0 impact on the run plan (tests). I'm only passing something since it's a mandatory field

Reply:
Then I suggest
```suggestion
            all_known_projects=set(),
```
to make it explicit that this field is not relevant to this test at all :)